engines/python/src/main/java/ai/djl/python/engine/PyModel.java

```diff
@@ -230,7 +230,7 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
         } else {
             int tensorParallelDegree = pyEnv.getTensorParallelDegree();
             if (tensorParallelDegree > 0) {
-                if (tensorParallelDegree > 1) {
+                if (getProperty("maxWorkers") == null && getProperty("gpu.maxWorkers") == null) {
```

Comment:
In MPI mode, I set maxWorker to 16

Reply:
in MPI mode, you can only restrict maxWorkers, you cannot set to 16